app/components/templates/regulars-info/regulars-info.component.tsx

```diff
@@ -0,0 +1,43 @@
+import {
+  BlockDivider,
+  Grid,
+  InfoBlock,
+  InfoSection,
+  SectionDividerContainer,
+} from './regulars-info.styles';
+import configData from '../../../config.json';
+import useMediaQuery from '../../../hooks/use-media-query';
+import {Divider} from '../../atoms/divider';
+import {Paragraph} from '../../atoms/paragraph';
+import {Subtitle} from '../../molecules/subtitle';
+
+const {regularsInfo, regularsDisclaimer} = configData;
+
+export const RegularsInfo = () => {
+  const isDesktop = useMediaQuery('(min-width: 768px)');
+
+  return (
+    <InfoSection>
+      <Grid>
+        {regularsInfo.map((info, index, arr) => {
+          const renderBlockDivider = isDesktop
+            ? index < arr.length - (2 - (arr.length % 2))
+            : index < arr.length - 1;
```

Comment:
What's this calculation for?

Reply:
On the original Figma design, on desktop mode, the last two InfoBlocks don't have a divider at the bottom. On handheld mode, it's only the last InfoBlock that doesn't have a divider, so:
- On desktop mode, the calculation checks whether the current element is one of the last two elements in the array if its length is even; otherwise, it checks if it's the last element. **If we ended up with, say, 3 InfoBlocks, only the last one wouldn't render the divider**.
- On mobile, it just checks whether the current element is the last element of the array.

So in summary it really just makes it so we can have a dynamic number of InfoBlocks.

Reply:
Im going to trust you on this one, because i don't know. 😄